lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/array_item_component.rb

```diff
@@ -0,0 +1,38 @@
+class ContentBlockManager::ContentBlockEdition::Details::Fields::ArrayItemComponent < ViewComponent::Base
```

Comment:
vv small nitpick, could these components be grouped in a folder together, as guess you can't use the item without the array?